src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs

```diff
@@ -336,5 +337,71 @@ private sealed class ImplBool : Scalar<bool>
                 public ImplBool(BinaryClassifierNoCalibration rec) : base(rec, rec._inputs) { }
             }
         }
+
+        /// <summary>
+        /// A reconciler for regression capable of handling the most common cases for regression.
+        /// </summary>
+        public sealed class MulticlassClassifier<TVal> : TrainerEstimatorReconciler
+        {
+            /// <summary>
+            /// The delegate to create the <see cref="MulticlassClassifier{TVal}"/> instance.
+            /// </summary>
+            /// <param name="env">The environment with which to create the estimator</param>
+            /// <param name="label">The label column name</param>
+            /// <param name="features">The features column name</param>
+            /// <param name="weights">The weights column name, or <c>null</c> if the reconciler was constructed with <c>null</c> weights</param>
+            /// <returns>A estimator producing columns with the fixed name <see cref="DefaultColumnNames.Score"/> and <see cref="DefaultColumnNames.PredictedLabel"/>.</returns>
+            public delegate IEstimator<ITransformer> EstimatorFactory(IHostEnvironment env, string label, string features, string weights);
+
+            private readonly EstimatorFactory _estFact;
+
+            /// <summary>
+            /// The general output for multiclass classifiers.
+            /// </summary>
+            public (Vector<float> score, Key<uint, TVal> predictedLabel) Output { get; }
+
+            protected override IEnumerable<PipelineColumn> Outputs => new PipelineColumn[] { Output.score, Output.predictedLabel };
+
+            private static readonly string[] _fixedOutputNames = new[] { DefaultColumnNames.Score, DefaultColumnNames.PredictedLabel };
+
+            /// <summary>
+            /// Constructs a new general regression reconciler.
+            /// </summary>
+            /// <param name="estimatorFactory">The delegate to create the training estimator. It is assumed that this estimator
+            /// will produce a single new scalar <see cref="float"/> column named <see cref="DefaultColumnNames.Score"/>.</param>
```

Comment:
>single new scalar <see cref="float"/> column named <see cref="DefaultColumnNames.Score"/> [](start = 31, length = 89)

The estimator I believe will produce two things, a vector float named `Score` and a key valued column named `PredictedLabel`, I think.